src/Nethermind/Nethermind.JsonRpc/Modules/Eth/EthRpcModule.cs

```diff
@@ -222,33 +223,47 @@ public Task<ResultWrapper<UInt256>> eth_getTransactionCount(Address address, Blo
     public ResultWrapper<UInt256?> eth_getBlockTransactionCountByHash(Hash256 blockHash)
     {
         SearchResult<Block> searchResult = _blockFinder.SearchForBlock(new BlockParameter(blockHash));
-        return searchResult.IsError
-            ? GetFailureResult<UInt256?, Block>(searchResult, _ethSyncingInfo.SyncMode.HaveNotSyncedBodiesYet())
-            : ResultWrapper<UInt256?>.Success((UInt256)searchResult.Object!.Transactions.Length);
+        if (searchResult.IsError)
+        {
+            return GetFailureResult<UInt256?, Block>(searchResult, _ethSyncingInfo.SyncMode.HaveNotSyncedBodiesYet());
+        }
+
+        return ResultWrapper<UInt256?>.Success((UInt256)searchResult.Object!.Transactions.Length);
```

Comment:
Keep shorter ternary operators!

Reply:
Leftovers after the initial implementation. Cleaned up in https://github.com/NethermindEth/nethermind/pull/8521/commits/cfa91ffd51fff1c8e9083884c7ec0db910eab4dd